cli/tests/e2e/e2e.test.ts

```diff
@@ -86,6 +86,7 @@ describe("e2e tests", function test() {
     });
 
     const user = new Keypair();
+    const deploySubsidy = true;
```

Comment:
here it should be false

Reply:
then should I also remove all `pollAddresses.subsidy` in the `verify` functions?
such as here: https://github.com/privacy-scaling-explorations/maci/blob/5f72f0a3447951ba6b9e0d554961f5752bbaca0f/cli/tests/e2e/e2e.test.ts#L158

Reply:
oh yea those shouldn't be there, good catch